### ISSUE TITLE
Is this project DEAD?

### ISSUE THREAD

**Issue by dpalic** [python-microservices/pyms#245]
**Describe the bug**
The project seems to have reasonable good pull requests and no replies on them.
So is there any maintainer who is around to give control to more developers to maintain this nice project?

**To Reproduce**
fetch latest version
run with latest python
project fails
reason => PR merges not done

**Expected behavior**
to accept the pull requests and to define more maintainers

**Comment by avara1986** [comment]
TL;DR: It's not dead, it's on vacation :upside_down_face: 

In recent years, I've been very busy with my "new" work at @DataDog and in my personal life as well. I've been wanting to pick up the project again for a while now, as I have many ideas in my head based on everything I've learned over these years that I want to incorporate into the project. However, I haven't found the right moment, and I had the perception that this project wasn't being very useful to many people, and if it's not useful, I wasn't very motivated to maintain it.

Thank you very much for your kind words, and I would love to accept anyone who is interested as a maintainer, discuss ideas, and keep the project alive. :+1: 

Thank you @hcamino for the heads up!
